airbyte-integrations/connector-templates/source-configuration-based/setup.py.hbs

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1.76",
+    "airbyte-cdk~=0.1.79",
```

Comment:
technically this doesn't exist yet, but I will merge https://github.com/airbytehq/airbyte/pull/15814 first which will should fix retrieving schemas while running from a job

Reply:
it now exists as of 8/22/22